src/land/smap_ssm2ioda.py

```diff
@@ -98,6 +102,8 @@ def _read(self):
             errs = errs[mask]
             qflg = qflg[mask]
             times = times[mask]
+            fill = (vals < valid_min) | (vals > valid_max) | (vals == _FillValue)
```

Comment:
should I take this out of the `if mask` conditional ???

Reply:
Maybe take it out of the `if self.mask` and only check for `fill = (vals == _FillValue)?!